MinimalClient/src/App.jsx

```diff
@@ -71,11 +82,13 @@ const TestCallContent = () => {
   } else {
     return (
       <>
-        <h5 className="card-title">Welcome {accounts[0].name}</h5>
-        <label>Enter a GUID Or Use Generated GUID for the call ID</label>
-        <form>
-          <input type="text" defaultValue={uuidv4()} id="guidTextBox" />
-        </form>
+        <h5 className="card-title">Welcome {accounts[0].name} </h5>
+        <h5 className="card-title">Access Token :&nbsp;&nbsp;
```

Comment:
Could you please rename it as AAD Access Token?

Reply:
Becasue we have AAD Access Token going into api requests as input and we are getting back ACS Access Token as response from apis. So using just Access Token is confusing.